diff --git a/nextstep/templates/Info.plist.in b/nextstep/templates/Info.plist.in
index 9338ea7f288..1f05788ded5 100644
--- a/nextstep/templates/Info.plist.in
+++ b/nextstep/templates/Info.plist.in
@@ -644,6 +644,8 @@ along with GNU Emacs.  If not, see <https://www.gnu.org/licenses/>.
 	<string>Emacs @version@ @copyright@</string>
 	<key>CFBundleIconFile</key>
 	<string>Emacs.icns</string>
+	<key>CFBundleIconName</key>
+	<string>EmacsLG3</string>
 	<key>CFBundleIdentifier</key>
 	<string>org.gnu.Emacs</string>
 	<key>CFBundleInfoDictionaryVersion</key>
@@ -685,15 +687,15 @@ along with GNU Emacs.  If not, see <https://www.gnu.org/licenses/>.
 		<key>NSAllowsArbitraryLoads</key>
 		<true/>
 	</dict>
-        <key>NSDesktopFolderUsageDescription</key>
-        <string>Emacs requires permission to access the Desktop folder.</string>
-        <key>NSDocumentsFolderUsageDescription</key>
-        <string>Emacs requires permission to access the Documents folder.</string>
-        <key>NSDownloadsFolderUsageDescription</key>
-        <string>Emacs requires permission to access the Downloads folder.</string>
-        <key>NSRemovableVolumesUsageDescription</key>
-        <string>Emacs requires permission to access files on Removable Volumes.</string>
-        <key>NSSpeechRecognitionUsageDescription</key>
-        <string>Emacs requires permission to handle any speech recognition.</string>
+	<key>NSDesktopFolderUsageDescription</key>
+	<string>Emacs requires permission to access the Desktop folder.</string>
+	<key>NSDocumentsFolderUsageDescription</key>
+	<string>Emacs requires permission to access the Documents folder.</string>
+	<key>NSDownloadsFolderUsageDescription</key>
+	<string>Emacs requires permission to access the Downloads folder.</string>
+	<key>NSRemovableVolumesUsageDescription</key>
+	<string>Emacs requires permission to access files on Removable Volumes.</string>
+	<key>NSSpeechRecognitionUsageDescription</key>
+	<string>Emacs requires permission to handle any speech recognition.</string>
 </dict>
 </plist>
